src/Calendar.Plugin.Sample/SampleApp/ViewModels/RangeSelectionPageViewModel.cs

```diff
@@ -1,114 +1,108 @@
-﻿using Xamarin.Plugin.Calendar.Models;
+﻿using Rg.Plugins.Popup.Services;
+using SampleApp.Model;
+using SampleApp.Views;
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
-using System.Threading.Tasks;
 using System.Linq;
-using System.Collections.ObjectModel;
+using System.Threading.Tasks;
 using System.Windows.Input;
 using Xamarin.Forms;
-using SampleApp.Model;
-using Rg.Plugins.Popup.Services;
-using SampleApp.Views;
+using Xamarin.Plugin.Calendar.Models;
 
 namespace SampleApp.ViewModels
 {
     public class RangeSelectionPageViewModel : BasePageViewModel, INotifyPropertyChanged
     {
-        public ICommand OpenRangePickerCommand => new Command(async () =>
-        {
-            await PopupNavigation.Instance.PushAsync(new CalendarRangePickerPopup(async (calendarPickerResult) =>
-            {
-                var message = "Calendar Range Picker Canceled!";
+        private DateTime? _endDate = DateTime.Today.AddDays(2);
 
-                if(calendarPickerResult.IsSuccess)
-                {
-                    var startDate = calendarPickerResult.SelectedDates[0];
-                    var endDate = calendarPickerResult.SelectedDates[calendarPickerResult.SelectedDates.Count - 1];
-                    message = $"Received date range from popup: {startDate:dd.MM.yyyy} - {endDate:dd.MM.yyyy}";
-                }
+        private DateTime _monthYear = DateTime.Today;
 
-                await App.Current.MainPage.DisplayAlert("Popup result", message, "Ok");
-            }));
-        });
+        private List<DateTime> _selectedDates = new List<DateTime> { };
```

Comment:
you can just use the new() keyword here, or at least call the constructor and not use an empty initializer.